packages/manager/apps/dedicated/client/app/dedicated/server/details/translations/Messages_fr_FR.json

```diff
@@ -1271,5 +1271,12 @@
   "server_error_vrack_detach": "Une erreur est survenue lors du détachement du réseau privé : {{error}}",
   "server_error_bandwidth_order": "Une erreur est survenue lors de la commande de bande passante : {{error}}",
   "server_configuration_task_INFRA_002_VIRTUAL_NETWORK_INTERFACE_GROUP": "Interfaces réseau de groupe",
-  "server_configuration_task_INFRA_002_VIRTUAL_NETWORK_INTERFACE_UNGROUP": "Dissocier les interfaces réseau"
+  "server_configuration_task_INFRA_002_VIRTUAL_NETWORK_INTERFACE_UNGROUP": "Dissocier les interfaces réseau",
+  "server_configuration_no_intervention": "Interventions sur site",
+  "server_configuration_on": "On",
+  "server_configuration_off": "Off",
+  "server_configuration_no_intervention_activate_success": "L'intervention sur site est activée sur le serveur {{serverName}}",
+  "server_configuration_no_intervention_activate_failed": "Une erreur est survenue lors de l'activation des interventions sur site sur le serveur {{serverName}}",
+  "server_configuration_no_intervention_deactivate_success": "L'intervention sur site est désactivé sur le serveur {{serverName}}",
```

Comment:
```suggestion
  "server_configuration_no_intervention_deactivate_success": "L'intervention sur site est désactivée sur le serveur {{serverName}}",
```

Reply:
I have sent for translations before your comment. If its a must fix for prod then I can make this change and reinitiate the translations.

Reply:
@radireddy Yes it's a typo. You can re-initiate a translation request.
Thank you!